tasks/broad/ImputationTasks.wdl

```diff
@@ -377,14 +377,59 @@ task PhaseAndImputeBeagle {
     String beagle_docker = "us-central1-docker.pkg.dev/morgan-fieldeng-gcp/imputation-beagle-development/imputation-beagle:0.0.1-01Mar24.d36-wip-temp-20240301"
     Int cpu = 8                    # This parameter is used as the nthreads input to Beagle which is part of how we make it determinstic.  Changing this value may change the output generated by the tool
     Int memory_mb = 32000          # value depends on chunk size, the number of samples in ref and target panel, and whether imputation is performed
-    Int xmx_mb = 27000             # I suggest setting this parameter to be 85-90% of the memory_mb parameter
+    Int xmx_mb = memory_mb - 5000             # I suggest setting this parameter to be 85-90% of the memory_mb parameter
     Int disk_size_gb = ceil(3 * size([dataset_vcf, ref_panel_bref3], "GiB")) + 10         # value may need to be adjusted
   }
 
   command <<<
     set -e -o pipefail
 
-    java -ea -XX:MaxRAMPercentage=90.0 -XX:-UseCompressedOops \
+    java -ea -Xmx~{xmx_mb}m \
+    -jar /usr/gitc/beagle.01Mar24.d36.jar \
```

Comment:
maybe for a future PR, but seems like maybe we should rename the jar something generic when creating the docker so we don't have to update multiple things when we update the Beagle version? alternatively if we like having this deliberately spelled out in the WDL, I'd buy that too.

Reply:
i _feel_ like this wold be something that rarely changes and we'd want to be deliberate about it so it might be ok to be explicit about the jar name?  I could go either way on this but I agree it should be a future change once we merge this wdl into WARP